helper/config.php

```diff
@@ -184,6 +184,15 @@ public function __construct() {
                   'hasMeta'            => false,
                   'addMetaAtStartOnly' => false,
                   'refresh'            => false);
+        // firsttemplatedefinitionwins
+        $this->config ['firsttemplatedefinitionwins'] =
+            array('value'              => NULL,
+                  'DWGlobalName'       => 'firsttemplatedefinitionwins',
```

Comment:
DokuWiki has not a setting with this name, so mentioning this here will have no use?

Reply:
Ouch, I guess you're right. I was mislead by the name `$this->config` and believed at the time of writing that the plugin configuration setting was made in a dynamic way, there. Instead, it looks like it's setting up the possible tags that can be used in the page syntax. Since my changes do not add any tags, the changes in helper/config.php can be dropped, obviously.